build.gradle.kts

```diff
@@ -0,0 +1,372 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import com.adarshr.gradle.testlogger.TestLoggerExtension
+import com.adarshr.gradle.testlogger.TestLoggerPlugin
+import org.jetbrains.intellij.IntelliJPluginExtension
+import org.jetbrains.intellij.tasks.PrepareSandboxTask
+import org.jetbrains.intellij.tasks.PublishTask
+import org.jetbrains.intellij.tasks.RunIdeTask
+import org.jetbrains.intellij.tasks.RunIdeForUiTestTask
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import software.aws.toolkits.gradle.IdeVersions
+import software.aws.toolkits.gradle.ProductCode
+import software.aws.toolkits.gradle.changelog.tasks.GenerateGithubChangeLog
+import software.aws.toolkits.gradle.findFolders
+import software.aws.toolkits.gradle.getOrCreate
+import software.aws.toolkits.gradle.resources.ValidateMessages
+
+buildscript {
+    repositories {
+        maven("https://plugins.gradle.org/m2/")
+        mavenCentral()
+        jcenter()
+    }
+    val kotlinVersion: String by project
+    val ideaPluginVersion: String by project
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
+        classpath("gradle.plugin.org.jetbrains.intellij.plugins:gradle-intellij-plugin:$ideaPluginVersion")
+        classpath("com.adarshr:gradle-test-logger-plugin:1.7.0")
+    }
+}
+
+val ideVersions = IdeVersions(project)
```

Comment:
Is there no better way than this?

Reply:
This is the way to do it, otherwise the value does not exist and the compiler fails (see docs: https://docs.gradle.org/current/userguide/kotlin_dsl.html#kotdsl:properties)